content/al/0006-produto-interno-ortogonalidade.md

```diff
@@ -0,0 +1,254 @@
+
+$$ proj_u v =\dfrac{ \langle u,v\rang }{\| u \|^2} $$
+:::
+
+:::info[Desigualdade Triangular]
+$$ \|u+v\|\le \|u\|+\|v\| $$
+:::
+
+### Complemento ortogonal
+
+Dois subespaços $U$ e $V$ dizem-se _subespaços complementares_ se qualquer vetor de $W$ se escreve na forma $w=u+v$ e se a interseção dos subespaços é nula ($U\cap V=\{\empty \}$).
+
+Tendo em conta esta definição,
+
+$dim W= dim U+ dim V$
+
+Pode-se expandir esta noção, criando a noção de
+
+:::info[Complemento Ortogonal]
+Seja $W$ um espaço linear munido de um produto intero e $S$ um subespaço de $W$
+O _complemento ortogonal_ de $S$ é o conjunto de todos os vetores de $W$ que são ortogonais a qualquer vetor de S. Designamos o complemento ortogonal do subespaço $S$ por $S^\perp$.
+:::
+
+:::info[Proposição]
+O complemento ortogonal $S^\perp$ do subespaço $S$ é um subespaço.
+:::
+
+:::info[Proposição]
+Seja $S$ um subespaço e $S^\perp$ o seu complemento ortogonal. Verifica-se que:
+
+$S \cap S^\perp=\{\empty\}$
+:::
+
+Tendo em conta que a interseção de um subespaço com o seu complemento ortogonal é o vazio, e tendo em conta que a sua união é o espaço, fica a questão de se qualquer vetor do espaço pode ser decomposto em vetores dos dois espaços. (Sim, e faz-se da seguinte forma)
+
+:::info[Teorema da decomposição ortogonal]
+Seja $W$ um espaço euclidiano e $S$ um subespaço de $W$. Qualquer vector $x\in W$ escreve-se de forma única como a soma de um vetor $x_S$ de $S$ com um vetor $x_{S^\perp}$ do complemento ortogonal de $S$. Isto é,
+
+$x=x_S+x_{S^\perp}$ com $x_S \in S$ e $x_{S^\perp}$
+
+Define-se a projeção ortognal de $x$ sobre o subespaço $S^\perp$ como $proj_{S^\perp}x=x_{S^\perp}$
+:::
+
+:::info[Hiperplano]
+Num espaço linear de dimensão $n$, chama-se _hiperplano_ a um subespaço de dimensão $(n-1)$
+:::
+
+:::info[Teorema da melhor aproximação]
+Sendo $W$ um espaço euclidiano e $S$ um subespaço de $W$ e $v$ um vetor de $W$, então
+
+$\| x-proj_Sx\|\le \|x-u\|$ para qualquer $u \in S$
+:::
+
+:::info[Distância a um subespaço]
+Seja $W$ um espaço linear, $S$ um subespaço de $W$ e $x$ um vetor de $W$. A distância de $x$ a $S$ é:
+
+$dist(x,S)= \|proj_{S^\perp}x\|$
+:::
+
+:::info[Ortogonalidade dos subespaços fundamentais de uma Matriz]
+$(EL(A))^\perp=N(A)$ e $(EC(A))^\perp=N(A^T)$
+:::
+
+### Ortogonalização de Gram-Schmidt
+
+Expressar vetores numa base ortornormada é relativamente simples, mas fica a questão de como obter uma tal base, a partir de um conjunto já existente de vetores. Para tal pode-se usar o método de ortogonalização de Gram Schmidt.
+
+:::info[Ortogonalização de Gram Schmidt]
+Seja $V=\{v_1,v_2,...,v_k\}$, com $k>1$, um conjunto linearmente independente de um espaço euclidiano. O conjutno $U=\{u_1,u_2,...,u_k\}$ formado pelos vetores
+
+$u_1=v_1$
+
+$u_2=v_2-\frac{\langle u_1,v_2\rang}{\|u_1\|^2}$
+
+$...$
+
+$u_k=v_k-proj_{u_1}v_k-proj_{u_2}v_k-...-proj_{u_{k-1}}v_k$
```

Comment:
Perhaps use [`\begin{array}` or `\being{aligned}`](https://katex.org/docs/supported.html#environments) here? I think the latter is more appropriate.

```suggestion
$$
\begin{aligned}
u_1 &= v_1 \\
u_2 &= v_2-\frac{\langle u_1,v_2\rang}{\|u_1\|^2} \\
    &= \dots \\
u_k &= v_k-\op{proj}_{u_1}v_k-\op{proj}_{u_2}v_k-\dots-\op{proj}_{u_{k-1}}v_k
\end{aligned}
$$
```

Reply:
Peço desculpa pela demora, só me consegui organizar agora para fazer as correções. Já deverá estar tudo. Sim, é de facto a minha primeira contribuição para um repositório externo.